src/API/Helpers/ApiClient.php

```diff
@@ -66,4 +67,29 @@ public function __call($name, $arguments) {
         return $builder->withHeaders($this->headers);
     }
 
+    /**
+     * Create a new RequestBuilder.
+     * Similar to the above but does not use a magic method.
+     *
+     * @param string $method - HTTP method to use (GET, POST, PATCH, etc).
+     *
+     * @return RequestBuilder
+     */
+    public function method($method) {
+        $method = strtolower( $method );
+        $builder = new RequestBuilder( [
+            'domain' => $this->domain,
+            'basePath' => $this->basePath,
+            'method' => $method,
+            'guzzleOptions' => $this->guzzleOptions
+        ] );
+        $builder->withHeaders($this->headers);
+
+        if ( in_array( $method, [ 'patch', 'post', 'put' ] ) ) {
+            $builder->withHeader( new ContentType('application/json') );
```

Comment:
I know this works for when you need to send a `body` in the request, but probably would be good to check if **all requests** send the `Accept` header so the server knows what you're expecting back in the response.

Reply:
I don't see an `Accept` header used in any of the examples in the docs. I'm wary of just dropping that in if it's not documented.